### PR TITLE
chore: add icon for NuGet packages

### PR DESCRIPTION
**Pull request by soi013**
It would be better to have an icon when searching for nuget.

image

Icon url was copied from "System.IO.Abstractions.Analyzers.csproj".

https://github.com/System-IO-Abstractions/System.IO.Abstractions.Analyzers/blob/develop/System.IO.Abstractions.Analyzers/System.IO.Abstractions.Analyzers.csproj

### CODE REVIEW

File: src/System.IO.Abstractions.TestingHelpers/System.IO.Abstractions.TestingHelpers.csproj [hunk 1]
```diff
@@ -4,6 +4,7 @@
         <RootNamespace>System.IO.Abstractions.TestingHelpers</RootNamespace>
         <Description>A set of pre-built mocks to help when testing file system interactions.</Description>
         <TargetFrameworks>net5.0;netstandard2.1;netstandard2.0;net461</TargetFrameworks>
+        <PackageIconUrl>https://avatars1.githubusercontent.com/u/40829877</PackageIconUrl>
```
Comment (fgreinacher) [inline src/System.IO.Abstractions.TestingHelpers/System.IO.Abstractions.TestingHelpers.csproj:7]: This property seems to be obsolete [according to the build log](https://github.com/System-IO-Abstractions/System.IO.Abstractions/pull/742/checks?check_run_id=3141881403#step:4:23). Could you change this to follow current guidelines?

**Review by fgreinacher — CHANGES_REQUESTED** [review]

**Comment by soi013** [comment]
OK. I will change it in two days after tomorrow.

**Comment by soi013** [comment]
Can I add the icon in the solution folder?
ex) /System.IO.Abstractions/System.IO.Abstractions_Icon_256x256.png

https://avatars1.githubusercontent.com/u/40829877

Is this logo image copyrighted by the System.IO.Abstractions project?

Because, PackageIcon needs folder-based source.

https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#packageicon
> Visual Studio supports PackageIcon for packages coming from a folder-based source.

**Comment by fgreinacher** [comment]
Thanks for driving this forward!

> Can I add the icon in the solution folder?
> ex) /System.IO.Abstractions/System.IO.Abstractions_Icon_256x256.png

Yes, please put it to `./images/icon_256x256.png`.

> Is this logo image copyrighted by the System.IO.Abstractions project?

Yes, it's from the [Arabia IO pack](https://www.shareicon.net/pack/arabia-io) that is free for personal use (see https://github.com/System-IO-Abstractions/System.IO.Abstractions/issues/324).

**Comment by fgreinacher** [comment]
Thanks a lot 👍

**Pull request merged**